isis/src/base/objs/Equalization/Equalization.truth

```diff
@@ -8546,13 +8546,13 @@ Testing loadInputs exception ...
 **USER ERROR** The input file [HoldList.lst] must contain at least 2 file names.
 
 Testing errorCheck exception (1)...
-**USER ERROR** Number of bands do not match between cubes [data/odyssey/testData/I00824006RDR.lev2.cub] and [data/base/testData/isisTruth.cub].
+**USER ERROR** Number of bands do not match between cubes [odyssey/testData/I00824006RDR.lev2.cub] and [/usgs/cpkgs/isis3/isis_data/base/testData/isisTruth.cub].
```

Comment:
I think this needs to be changed to not have the full path for the test to succeed.

Reply:
Is this missing a regex to strip out the path, or is the regex not working?